README.md

```diff
@@ -97,11 +95,7 @@ By default metrics are gathered once per second.
 
 Example of running Snap ceph perf counters collector and writing data to file.
 
-Run the Snap daemon on each node with defaults settings:
-```
-$ $SNAP_PATH/snapteld -l 1 -t 0
-```
-Or set custom settings in Snap Global Config in Ceph section (see examples/configs/snap-config-sample.json):
```

Comment:
Right, I will replace that with

Run snap daemon with default config values (see the table above):

```
$ $SNAP_PATH/snapteld -l 1 -t 0
```

Or with your custom Snap Global Config (see examples/configs/snap-config-sample.json):

```
$ $SNAP_PATH/snapteld -l 1 -t 0 --config $SNAP_CEPH_PLUGIN_DIR/examples/configs/snap-config-sample.json
```